.env.default

```diff
@@ -3,8 +3,24 @@ CLUSTER=conn
 GIT_REPO_NAME="ce-connector-template"
 GIT_BASE_BRANCH=main
 
-APP_PORT=3000
-APP_ENV=production
-
 # External ports used in dev docker-compose.yml
 HOST_APP_PORT=3000
+
+# app
+APP_VERSION=1
+GIT_COMMIT_SHA=sha
```

Comment:
How we're going to use it, and should we automatically update it?

Reply:
This will be exposed as a part of a public healthcheck endpoint (to be added in coming PRs). I'll look up how we populate it in our other services and try to include the logic in the repo as well.